modules/build/src/test/scala/scala/build/tests/InputsTests.scala

```diff
@@ -1,8 +1,30 @@
 package scala.build.tests
 
 import com.eed3si9n.expecty.Expecty.expect
+import scala.build.Build
+import scala.build.Inputs
+import scala.build.options.{BuildOptions, InternalOptions, MaybeScalaVersion}
+import scala.build.tests.util.BloopServer
+import scala.build.{BuildThreads, Directories, LocalRepo}
 
 class InputsTests extends munit.FunSuite {
+  val buildThreads    = BuildThreads.create()
+  val extraRepoTmpDir = os.temp.dir(prefix = "scala-cli-tests-extra-repo-")
+  val directories     = Directories.under(extraRepoTmpDir)
+  def bloopConfigOpt  = Some(BloopServer.bloopConfig)
+  val baseOptions = BuildOptions(
+    internal = InternalOptions(
+      localRepository = LocalRepo.localRepo(directories.localRepoDir),
+      keepDiagnostics = true
+    )
+  )
+  def sv2 = "2.13.5"
+  val defaultOptions = baseOptions.copy(
```

Comment:
It will be good enough if you use `baseOptions` in your tests. You don't have to specify Scala version.